tutorial/global-context-item/stylesheet_testable.xsl

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet exclude-result-prefixes="#all" version="3.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<!--
+		This stylesheet is a testable version of stylesheet_original.xsl.
+	-->
+
+	<!-- @use has been changed to "optional" as a workaround for XSpec. -->
+	<xsl:global-context-item as="document-node(element(foo))" use="optional" />
+
+	<!-- This global variable is a workaround for XSpec. In XSpec document, you must override this
+		global variable with a global x:variable pointing to a source document. -->
```

Comment:
Why `x:variable` instead of `x:param`? The [wiki](https://github.com/xspec/xspec/wiki/Writing-Scenarios#global-parameters) documents `x:param` for overrides of both global XSLT parameters and global XSLT variables, and documents `x:variable` for test-specific variables. I have the same question about the wiki updates you referred to in this PR description.

If you think it should be equally acceptable to override a global XSLT variable using either `x;param` or `x:variable`, then it would be useful to clarify the wiki. Maybe an extra sentence in https://github.com/xspec/xspec/wiki/Writing-Scenarios#xspec-variables would suffice.

Reply:
Good point. For consistency and with `run-as=external` in mind, `x:param` will be better. I updated this pr in cdd89b5d7c1ffc313ef4bcd343ea80f1db21b981 and also [Wiki](https://github.com/xspec/xspec/wiki/Writing-Scenarios#global-context-item) ([diff](https://github.com/xspec/xspec/wiki/Writing-Scenarios/_compare/473476b40da315b99d5ad294846c3a6f67caf6d8...d5ba27127e8be1173b8cb46accb7942ca5494afe)). Can you review the change?

Reply:
These latest changes look good.